server/config/summaries/place_summaries.json

```diff
@@ -0,0 +1,152 @@
+ },
+ "geoId/0820000": {
+  "summary": "Denver, Colorado is the most populous city in the state and the 19th most populous city in the United States. The population of Denver has increased from 467,549 in 1990 to 711,463 in 2021. The median individual income in Denver is $45457, which is ranked 66 of 217 cities in Colorado and 3043 of 15229 cities in the United States. The median age in Denver was 34.8 in 2021."
+ },
+ "geoId/4055000": {
+  "summary": "Oklahoma City, OK is the most populous city in Pottawatomie County, Oklahoma County, Cleveland County, Canadian County, and Oklahoma. The population of Oklahoma City has increased from 444605 in 1990 to 687725 in 2021. The median individual income in Oklahoma City is $34017, which is higher than the median income of all cities in the United States. The median age in Oklahoma City was 34.9 in 2021."
+ },
+ "geoId/2447875": {
+  "summary": "Lonaconing, Maryland is a city with a population of 989. It is the 4th most populous city in Allegany County and the 99th most populous city in Maryland. The median individual income in Lonaconing is $26,181, which is ranked 4th of 8 cities in Allegany County, 307th of 320 cities in Maryland, and 11628th of 15229 cities in the United States. The median age in Lonaconing is 36.7 years, which is lower than the median age of 39.7 years in Allegany County and 42.2 years in Maryland."
+ },
+ "geoId/1755171": {
+  "summary": "Odell, Illinois is a small town with a population of 989. It is located in Livingston County and is ranked 8th out of 8 cities in the county. The median individual income is $28,000 and the median age is 36.4 years."
+ },
+ "geoId/4052600": {
+  "summary": "North Enid, Oklahoma is the 3rd most populous city in Garfield County with a population of 987 as of 2021. It is ranked 10271st in the United States. The median individual income is $27146, which is ranked 1st of 3 cities in Garfield County, 35th of 229 cities in Oklahoma, and 6613th of 15229 cities in the United States. The median age in North Enid was 44.0 in 2011 and has increased to 44.9 in 2021. North Enid is ranked 2 of 3 cities in Garfield County, 23 of 247 cities in Oklahoma, and 3932 of 16433 cities in the United States."
+ },
+ "geoId/2874760": {
+  "summary": "Tunica, MS is the 2nd largest city in Tunica County with a population of 987 in 2021. The population has fluctuated between 1000 and 1200 people since 1990. Tunica has a median individual income of $29522, which is ranked 2nd in Tunica County, 66th in Mississippi, and 11333rd in the United States. The median age in Tunica is 45.2 years old, which is ranked 1st in Tunica County, 8th in Mississippi, and 1940th in the United States."
+ },
+ "geoId/5531300": {
+  "summary": "Green Lake, Wisconsin is a small city with a population of 985. It is located in Green Lake County and has a median individual income of $28,558. The median age in Green Lake is 47.5 years old, which is lower than the national average of 38.1 years old."
```

Comment:
error: " 47.5 years old, which is lower than the national average of 38.1 years old."